doc/release-notes.md

```diff
@@ -1,9 +1,8 @@
-# Dash Core version v21.1.0
+# Dash Core version v21.1.1
 
-This is a new minor version release, bringing important bugfixes.
+This is a new patch version release, bringing important bugfixes.
 
-This release is **mandatory** for all masternodes.
-This release is optional but recommended for all other nodes.
+This release is **optional** but recommended for all nodes.
```

Comment:
```suggestion
This release is **mandatory** for all nodes that serve 3rd (client's) party RPC calls.
This release is **optional** but recommended for all other nodes.
```

because they actually exists: https://www.dashcentral.org/p/DCD_RPC-Web-Proxy-and-Explorer_2024-Q3

Reply:
I mean, it's not mandatory, strictly speaking it is optional.